GitUI/UserControls/RevisionGrid/Columns/MessageColumnProvider.cs

```diff
@@ -144,15 +144,12 @@ public override bool TryGetToolTip(DataGridViewCellMouseEventArgs e, GitRevision
                 return true;
             }
 
-            if (revision.IsArtificial)
+            // Artificial
+            var stats = _grid.GetChangeCount(revision.ObjectId);
```

Comment:
Could you please align the variable names - here it's called `stats`, below it's called `changes`. Also please use type name here instead of `var` as the type isn't obvious here.

Reply:
OK, I did not change the names in existing code.